configmap-puller/cmd/configmap-puller/main.go

```diff
@@ -0,0 +1,164 @@
+// Note: the example only works with the code within the same release/branch.
```

Comment:
ups - a copy/paste artifact i forgot to remove! :sweat_smile:

Reply:
oops missed out on that one! Done